tests/link/test_numba.py

```diff
@@ -217,7 +217,7 @@ def assert_fn(x, y):
     numba_res = aesara_numba_fn(*inputs)
 
     # Get some coverage
-    eval_python_only(fn_inputs, fgraph, inputs)
+    # eval_python_only(fn_inputs, fgraph, inputs)
```

Comment:
For some reason the `eval_python_only` throws error related to `Type` even though it just there for Pythonic coverage. Is this because of the changed mock keys (`numa.njit` to `numba_njit`)?

Reply:
If you add an option to `numba_njit`, you'll probably need to account for it in the mock for `numba_njit`, as well.

Reply:
I tried mock patching numba_njit directly, it says that it cannot find a target for patch. I guess we'd have to keep the current patch target i.e. numba.njit find suitable implementation for that. I'm not exactly familiar with how this patching works, so having a bit of trouble coming up with one.